### PR TITLE
started work on the integration of QR codes

### LINKED ISSUE #50: Create 'QRCode' component

### LINKED ISSUE #51: Slide in a QR code for a presentation request

### PR DESCRIPTION
**Pull request by Vanderscycle**
Working on issue #50 and #51

### CODE REVIEW

**Comment by Vanderscycle** [comment]
Main issue with this pr is that the css isn't lining up properly.

File: frontend/package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "svelte": "^3.37.0",
     "svelte-check": "^2.1.0",
     "svelte-preprocess": "^4.7.4",
+    "svelte-qrcode": "^1.0.0",
```
Comment (brianorwhatever) [inline frontend/package.json]: lets remove this dependency and add qrious directly

Comment (brianorwhatever) [reply frontend/package.json]: oh it looks like you accidentally added it to the root package.json. i did that recently too

**Review by brianorwhatever — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (Vanderscycle) [reply frontend/package.json]: solve with ea95042

**Pull request merged**